apps/web/src/app/vault/individual-vault/view.component.html

```diff
@@ -0,0 +1,33 @@
+<bit-dialog dialogSize="default">
+  <span bitDialogTitle>
+    {{ cipherTypeString }}
+  </span>
+  <ng-container bitDialogContent>
+    <app-cipher-view [cipher]="cipher"></app-cipher-view>
+  </ng-container>
+  <ng-container bitDialogFooter>
+    <button
+      bitButton
+      bitFormButton
+      (click)="edit()"
+      buttonType="primary"
+      type="button"
+      [disabled]="!cipher.edit"
+    >
+      {{ "edit" | i18n }}
+    </button>
+    <div class="ml-auto">
+      <button
+        bitButton
+        bitFormButton
+        type="button"
+        buttonType="secondary"
+        class="btn-outline-danger"
```

Comment:
`buttonType="danger"` should be able to handle all styling of a deletion here rather than adding a class

```suggestion
        buttonType="danger"
```